src/lib/getSecurityUi.ts

```
@@ -0,0 +1,33 @@
+  if (!security) {
+    return securityUi
+  }
+
+  security.forEach((securityOption) => {
+    const andItems = {
+      id: Object.keys(securityOption).length ? Object.keys(securityOption).join('|') : NO_SECURITY,
+      schemes: {},
+    } as SecurityUiItem
+
+    Object.keys(securityOption).forEach((key) => {
+      if (securitySchemes && securitySchemes[key]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)